pkgs/top-level/all-packages.nix

```diff
@@ -1440,6 +1440,9 @@ with pkgs;
 
   winetricks = callPackage ../applications/emulators/wine/winetricks.nix { };
 
+  bigpemu = callPackage ../by-name/bi/bigpemu/package.nix {
+    buildFHSUserEnv = buildFHSEnv;
+  };
```

Comment:
```suggestion
```
I don't think you need to do `all-packages` anymore with the `by-name` system.

Reply:
Artifact of before I realized I needed to do the by-name. Updated.